spec/ruby-enum/enum_spec.rb

```diff
@@ -169,4 +169,13 @@ class EmptyEnums
       expect { EmptyEnums::ORANGE }.to raise_error Ruby::Enum::Errors::UninitializedConstantError
     end
   end
+
+  # https://github.com/dblock/ruby-enum/pull/11
```

Comment:
I don't think this comment is useful, it's pretty clear what's going on. CHANGELOG and `git blame` can always help you find the source.